src/plugin/withStripe.ts

```diff
@@ -72,6 +74,22 @@ export function setApplePayEntitlement(
   return entitlements;
 }
 
+/**
+ * Add a blank Swift file to the Xcode project for Swift compatibility.
+ */
+export const withNoopSwiftFile: ConfigPlugin = (config) => {
```

Comment:
@cruzach does this add the needed bridging header?

Reply:
the bridging header is added by default by Expo 👍